packages/app-bridge/src/AppBridgeTheme.ts

```diff
@@ -61,6 +61,12 @@ export type ThemeCommand = CommandNameValidator<
     Pick<CommandRegistry, 'closeAssetChooser' | 'openAssetChooser' | 'openNavigationManager'>
 >;
 
+export type TemplateContext = { templateId?: string; type: ThemeTemplate } & (
```

Comment:
```suggestion
export type TemplateContext = { templateId: string; type: ThemeTemplate } & (
```
Should always be defined (right now is `default`)